src/components/dialogmoteikkeaktuell/DialogmoteikkeaktuellSkjema.tsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import { Link, Navigate } from "react-router-dom";
+import { moteoversiktRoutePath } from "@/routers/AppRouter";
+import { useDialogmotekandidat } from "@/data/dialogmotekandidat/dialogmotekandidatQueryHooks";
+import {
+  CreateIkkeAktuellDTO,
+  IkkeAktuellArsak,
+} from "@/data/dialogmotekandidat/types/dialogmoteikkeaktuellTypes";
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import { useSettDialogmoteikkeaktuell } from "@/data/dialogmotekandidat/useSettDialogmoteikkeaktuell";
+import { SkjemaInnsendingFeil } from "@/components/SkjemaInnsendingFeil";
+import { SubmitHandler, useForm } from "react-hook-form";
+import {
+  Alert,
+  Box,
+  Button,
+  Radio,
+  RadioGroup,
+  Textarea,
+} from "@navikt/ds-react";
+
+export const texts = {
+  noBrev: "Det blir ikke sendt ut brev.",
+  infoKandidatlist: `Når du setter ikke aktuell vil arbeidstakeren bli fjernet fra kandidatlisten. Dersom du på et senere tidspunkt vurderer at det likevel er nødvendig med et dialogmøte, kan du kalle inn til dialogmøte ved å søke deg frem til denne arbeidstakeren.`,
+  arsakLegend: "Årsak til ikke aktuell (obligatorisk)",
+  arsakErrorMessage: "Vennligst angi årsak.",
+  beskrivelseLabel: "Beskrivelse (valgfri)",
+  send: "Sett ikke aktuell",
+  avbryt: "Avbryt",
+};
+
+export interface IkkeAktuellArsakText {
+  arsak: IkkeAktuellArsak;
+  text: string;
+}
+export const ikkeaktuellArsakTexts: IkkeAktuellArsakText[] = [
+  {
+    arsak: IkkeAktuellArsak.ARBEIDSTAKER_AAP,
+    text: "Innbygger mottar AAP",
+  },
+  {
+    arsak: IkkeAktuellArsak.ARBEIDSTAKER_DOD,
+    text: "Innbygger er død",
+  },
+  {
+    arsak: IkkeAktuellArsak.DIALOGMOTE_AVHOLDT,
+    text: "Dialogmøte avholdt",
+  },
+];
+
+export const dialogmoteikkeaktuellSkjemaBeskrivelseMaxLength = 2000;
+
+export interface DialogmoteikkeaktuellSkjemaValues {
+  arsak: IkkeAktuellArsak;
+  beskrivelse?: string;
+}
+
+const DialogmoteikkeaktuellSkjema = () => {
+  const personIdent = useValgtPersonident();
+  const { isKandidat } = useDialogmotekandidat();
+  const settDialogmoteikkeaktuell = useSettDialogmoteikkeaktuell();
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<DialogmoteikkeaktuellSkjemaValues>();
+
+  if (!isKandidat || settDialogmoteikkeaktuell.isSuccess) {
+    return <Navigate to={moteoversiktRoutePath} />;
+  }
+
+  const onSubmit: SubmitHandler<DialogmoteikkeaktuellSkjemaValues> = (
+    values
+  ) => {
+    const newIkkeAktuell: CreateIkkeAktuellDTO = {
+      personIdent: personIdent,
+      arsak: values.arsak,
+      beskrivelse: values.beskrivelse,
+    };
+    settDialogmoteikkeaktuell.mutate(newIkkeAktuell);
+  };
+
+  return (
+    <Box background="surface-default" padding="8">
+      <Alert variant="info" size="small" className="p-4 mb-4">
+        {texts.noBrev}
+      </Alert>
+      <p>{texts.infoKandidatlist}</p>
```

Comment:
Mulig vi vil bruke BodyShort her https://aksel.nav.no/komponenter/core/typography#02e18c03b059

Reply:
Det er egentlig bare å endre til dette og importere, men det må kanskje gjøres i Unntak-siden også?
```typescript
<BodyShort size="small">{texts.infoKandidatlist}</BodyShort>
```

Hvis du ender med å endre begge, ville jeg byttet `<Box background="surface-default" padding="8">` til `<Box background="surface-default" padding="6">` på begge sider også, siden det er det som brukes på tilsvarende side for innkalling av dialogmøte.

Reply:
Spacing'en rundt tekstboksen ble helt feil, så endre tilbake til <p>...</p>. Noen som kan dette bedre enn meg får evnt gjøre det etterpå.